WalletWasabi/WabiSabi/Backend/WabiSabiConfig.cs

```diff
@@ -114,6 +114,9 @@ public WabiSabiConfig(string filePath) : base(filePath)
 	[JsonProperty(PropertyName = "IsCoinVerifierEnabled", DefaultValueHandling = DefaultValueHandling.Populate)]
 	public bool IsCoinVerifierEnabled { get; set; } = false;
 
+	[JsonProperty(PropertyName = "RiskFlags", DefaultValueHandling = DefaultValueHandling.Populate)]
+	public List<int>? RiskFlags { get; set; } = new();
```

Comment:
How does this look like in the file? Easy to edit manually?

Reply:
```
"RiskFlags": [
    1,
    25,
    56
  ],
```

Reply:
That's fine.